falcon/media_serializer.py

```diff
@@ -0,0 +1,11 @@
+from typing import Dict
+from typing import Union
+
+from falcon.link import Link
+
+
+class Serializer:
```

Comment:
this is to avoid using protocol correct?

I think we should leave a note here.


also move to typing?

Reply:
Yes this is to avoid Protocol, I cannot move to typing because circular import :(